backend/sessions.ts

```diff
@@ -11,10 +11,12 @@ export type Item = {
   added: number;
 };
 
+// TO ADD: In the session, clientName should also be saved together with clientId
 export interface SessionData {
   code: string;
   hostId: string;
   clientIds: Set<string>;
+  clientIdsWithNick: Map<string, string>;
```

Comment:
I've noticed that `Sets` and `Maps` don't seem to work to well with UIX right now.

How about we just combine cliendIds and the nick and potently other stuff we want to save for each user by doing something like this:

```js
export interface Client {
	id: string
	name: string
}

// and the in the SessionData:

clients: Record<string, Client>; // dictionary with client ids as key and client data as value
```

Reply:
Alright that makes sense, i will then add Client Interface to SessionData